data/GalaxyStellarMassGasMetallicity/conversion/convertStrom2022.py

```diff
@@ -0,0 +1,80 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+# Meta-data
+comment = "The median stellar mass is 1e10 Msun."
+citation = "Strom et al. (2022)"
+bibcode = "2022ApJ...925..116S"
+plot_as = "line"
+redshift = np.array([2.0, 2.7])
```

Comment:
Let's expand the range a little bit, so whenever we make plots at z \approx 2 or z \approx 3, these data are always included.
```suggestion
redshift = np.array([1.9, 3.1])
```